src/api/firebase.js

```diff
@@ -77,3 +77,5 @@ export async function deleteItem() {
 	 * this function must accept!
 	 */
 }
+
+export async function checkToken() {}
```

Comment:
Here it seems to be that you have imported getDocs and the checkToken function we created but it contains no data. What did you want to do here?
The function should be:
```
export async function checkToken(tokenName = '') {
	const collectionRef = collection(db, tokenName);

	const snapshot = await getDocs(collectionRef);

	return !!snapshot.docs.length;
}
```

Otherwise it will conflict with ours

Reply:
We didn't touch this at all. :)

Reply:
Added the correct function!